src/main/java/it/gov/pagopa/admissibility/rest/anpr/residence/ResidenceAssessmentRestClientImpl.java

```diff
@@ -101,7 +115,7 @@ private RichiestaE002DTO generateRequest(String fiscalCode) {
         TipoCriteriRicercaE002DTO criteriRicercaE002DTO = new TipoCriteriRicercaE002DTO()
                 .codiceFiscale(fiscalCode);
 
-        String dateWithoutHoursString = new  SimpleDateFormat("yyyy-MM-dd").format(dateNow);
+        String dateWithoutHoursString = new SimpleDateFormat("yyyy-MM-dd").format(dateNow);
```

Comment:
let's store new SimpleDateFormat("yyyy-MM-dd") into a const variable of this class